cmd/umts-dev-setup.rb

```diff
@@ -6,13 +6,15 @@
 
 require_relative '../lib/nodenv_setup'
 require_relative '../lib/rbenv_setup'
+require_relative '../lib/brew_path_setup'
 
 module Homebrew
   module_function
 
   def umts_dev_setup
     setup_nodenv
     setup_rbenv
+    setup_brew_path
```

Comment:
This needs to happen first becuase the `rbenv` setup launches subshells and expects to be able to run "`rbenv`"